tests/test_wave_01.py

```diff
@@ -119,12 +119,12 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
+    #raise Exception("Test needs to be completed.")
```

Comment:
We want to make sure we are finishing up these assertions! An example here would be:

     assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
     assert updated_data["watched"][0]["genre"] == GENRE_1
     assert updated_data["watched"][0]["rating"] == RATING_1